client/pages/sections/protocols/protocol-sections.js

```diff
@@ -46,7 +47,9 @@ class ProtocolSections extends PureComponent {
       editable,
       newComments,
       readonly,
-      schemaVersion
+      schemaVersion,
+      latest,
+      granted
```

Comment:
same here, we should be getting these props in the downstream component - `ChangedBadge`